compiler/src/iree/compiler/Dialect/Stream/Transforms/EncodeTensors.cpp

```diff
@@ -335,12 +336,38 @@ static Value canonicalizeFillPattern(Value pattern, PatternRewriter &rewriter) {
 
   // Get floats into integer form.
   auto patternType = pattern.getType();
-  unsigned bitWidth = patternType.getIntOrFloatBitWidth();
+  unsigned bitWidth = 0;
+  if (isa<ComplexType>(patternType)) {
+    bitWidth = dyn_cast<ComplexType>(patternType)
```

Comment:
We should probably make `bitWidth` match the full size (e.g. `2 *  elementsize`). This avoids the complex check further down.